java/integTest/src/test/java/glide/SharedCommandTests.java

```diff
@@ -1055,6 +1057,105 @@ public void zcard(BaseClient client) {
         assertTrue(executionException.getCause() instanceof RequestException);
     }
 
+    @SneakyThrows
+    @ParameterizedTest
+    @MethodSource("getClients")
+    public void zrandmember(BaseClient client) {
+        String key1 = UUID.randomUUID().toString();
+        String key2 = UUID.randomUUID().toString();
+        Map<String, Double> membersScores = Map.of("one", 1.0, "two", 2.0);
+        assertEquals(2, client.zadd(key1, membersScores).get());
+
+        String randMember = client.zrandmember(key1).get();
+        assertTrue(randMember.equals("one") || randMember.equals("two"));
```

Comment:
why? if it is caused by random map ordering, use `LinkedHashMap`

Reply:
Random map ordering? Confused on what you mean. We receive a random element from the sorted set on Redis hence I need to be able to check if the key is correct?

Reply:
Oh right
Please ignore